Source/JavaScriptCore/wasm/js/WebAssemblyModuleRecord.cpp

```diff
@@ -292,6 +292,7 @@ void WebAssemblyModuleRecord::initializeImports(JSGlobalObject* globalObject, JS
                             m_instance->setGlobal(import.kindIndex, value);
                         } else {
                             value = Wasm::internalizeExternref(globalValue->global()->get(globalObject));
+                            RETURN_IF_EXCEPTION(scope, void());
```

Comment:
internalizeExternref can throw? It didn't look like it to me. If so, can we get more of these checks around the other uses?

Reply:
Yeah, we should do that otherwise you'll be passing a potentially garbage value into `internalizeExternref`

As a nit: I would also name the variable `global` rather than `gotValue` it's a bit more specific.